src/language/typescript/3.0/serializers/operation-object.ts

```diff
@@ -247,6 +259,32 @@ export const serializeOperationObject = combineReader(
 
 		const serializedResponses = serializeResponsesObject(from)(operation.responses);
 
+		const responseType: XHRResponseType = pipe(
+			lookup('200', operation.responses),
+			chain(response => e.deepLookup(response, ResponseObjectCodec, ReferenceObjectCodec)),
+			chain(response => response.content),
+			chain(content => lookup('application/json', content)),
```

Comment:
This looks weird - how is it even possible for a `binary` response to be an `application/json`?

Reply:
fixed